benchmark/collective/resnet/utils/fp16_utils.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import print_function
```

Comment:
what's the different between this fp16_utils and https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/contrib/mixed_precision/fp16_utils.py

Reply:
the file is originally from paddle/model. The only difference is that the append_cast_op function is deleted and cast_fp16_to_fp32 and cast_fp32_to_fp16 functions are added.